libs/partners/google-genai/langchain_google_genai/llms.py

```diff
@@ -151,6 +151,18 @@ def lc_secrets(self) -> Dict[str, str]:
     def _model_family(self) -> str:
         return GoogleModelFamily(self.model)
 
+    @property
+    def _identifying_params(self) -> Dict[str, Any]:
```

Comment:
which method that is not overriden will use this property?

Reply:
There is no much documentation about this property. At first I thought  it was used for serialization but is not really only that. When working with callbacks this property is used somehow to map the invocation parameters.
WIthout it you can't get the model name in callbacks events. 
Check:
https://github.com/search?q=org%3Alangchain-ai+_identifying_params&type=code